tasks/src/depth-estimation/data.ts

```diff
@@ -0,0 +1,37 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [],
+	demo: {
+		inputs: [
+			{
+				filename: "depth-estimation-input.jpg",
+				type:     "img",
+			},
+		],
+		outputs: [
+			{
+				filename: "depth-estimation-output.png",
+				type:     "img",
+			},
+		],
+	},
+	metrics: [],
+	models: [
+		{
+			// TO DO: write description
+			description: "Strong Depth Estimation model trained on 1.4 million images.",
+			id:          "Intel/dpt-large",
+		},
+		{
+			// TO DO: write description
+			description: "Strong Depth Estimation model trained on the KITTI dataset.",
+			id:          "vinvino02/glpn-kitti",
+		},
+	],
+	summary:      "Depth estimation is the task of predicting depth of the objects present in an image.",
+	widgetModels: ["Intel/dpt-large"],
```

Comment:
As of now we don't have a widget for this model.
```suggestion
	widgetModels: [""],
```

Reply:
Why?

Reply:
We don't have the widget for the model.
<img width="807" alt="Ekran Resmi 2022-11-21 18 01 55" src="https://user-images.githubusercontent.com/53175384/203087590-c84ac4be-b89c-4749-8a4b-199a70ddd174.png">